app/views/content/dates_and_deadlines.md

```diff
@@ -1,9 +1,22 @@
+### <%= CycleTimetable.cycle_year_range %> recruitment cycle – current
 | **Date and time** | **What happens** |
 | --- | --- |
-| <%= CycleTimetable.apply_opens.to_fs(:govuk_date_and_time) %> | Start of recruitment cycle - candidates can apply for courses. |
+| <%= CycleTimetable.apply_opens.to_fs(:govuk_date_and_time) %> | Start of <%= CycleTimetable.cycle_year_range %> recruitment cycle - candidates can apply for courses. |
 | <%= holidays[:christmas][:begins].to_fs(:govuk_date) %> to <%= holidays[:christmas][:ends].to_fs(:govuk_date) %> | This period is not counted as working days when calculating time to make a decision. |
 | <%= holidays[:easter][:begins].to_fs(:day_and_month) %> to <%= holidays[:easter][:ends].to_fs(:govuk_date) %> | This period is not counted as working days when calculating time to make a decision. |
-| 1 July 2022 | Time to make a decision is reduced from 40 working days to 20 working days. |
+| 1 July <%= CycleTimetable.current_year %> | Time to make a decision is reduced from 40 working days to 20 working days. |
```

Comment:
Can this be the same day as the summer recruitment banner appears?

Reply:
If so you could use `CycleTimetable.show_summer_recruitment_banner`

Reply:
I think these are the same date, so I've updated it in 488ec682ff978235e773093a168fbab4ef4f9d45 - had to add an extra class method. @tomas-stefano does that look ok?

Reply:
Sounds good 🚀 🚢